classes/Contact.php

```diff
@@ -70,7 +70,7 @@ class ContactCore extends ObjectModel
                 'size' => 255,
             ],
             'description' => [
-                'type' => self::TYPE_STRING,
```

Comment:
Right now constraints of the form don't make sense. There is validation isCleanHtml here and in formType but because it's type is TYPE_STRING here it means it won't even save simple HTML like <div>. I don't know if modifying old object is "allowed" in this case.

If it's not then I will change constraints to reflect that it won't actually save HTML.

Reply:
Hi @JevgenijVisockij it seems you have found an inconsistency!

Modifying old object is allowed when we find an inconsistency, it's even a good thing! The tricky part is to find out
- is the type right (so it's a string) and the constraints wrong?
or
- is the type wrong (so it's a HTML field) and the constraints right?

I tested a bit and I think that this field is actually indeed a string. For me the constraints are wrong 🤔  . Because
- when I used current form to save HTML the markup is removed to transform it into plain string
- the form input is not a HTML WYSIWYG and I dont expect merchants to write hardcoded HTML

Reply:
Changes needed

Reply:
"when I used current form to save HTML the markup is removed to transform it into plain string" 

This happens exactly because of `'type' => self::TYPE_STRING,` so it was a question whether the object model was right or form is right. 

`the form input is not a HTML WYSIWYG and I dont expect merchants to write hardcoded HTML`

I agree here, I am gonna the constraints so that it won't actually allow you to input HTML.